src/resources/nfts/simplehash/utils.ts

```diff
@@ -131,7 +132,10 @@ export function simpleHashNFTToUniqueAsset(nft: SimpleHashNFT): UniqueAsset {
   const standard = nft.contract.type;
 
   return {
-    animation_url: nft?.video_url,
+    animation_url: maybeSignUri(
+      nft?.video_url ?? nft.extra_metadata?.animation_original_url ?? undefined,
```

Comment:
wonder if we should include this in `handleAndSignImages`

Reply:
Can we even feed videos into cloudinary?